build/script/init_agent

```diff
@@ -65,43 +65,46 @@ DEFAULT_BRIDGE=`cat ${AGENT_CONFIG_PATH} | grep datapathConfig: -A1 | grep -v da
 LOCAL_GW_IP=`cat ${AGENT_CONFIG_PATH} | grep localGwIP | awk -F ':' '{print $2}' | awk '$1=$1'`
 GW_IFACE=${DEFAULT_BRIDGE}-gw
 GW_LOCAL_IFACE=${DEFAULT_BRIDGE}-gw-local
+POLICY_BRIDGE="${DEFAULT_BRIDGE}-policy"
+CLS_BRIDGE="${DEFAULT_BRIDGE}-cls"
+UPLINK_BRIDGE="${DEFAULT_BRIDGE}-uplink"
 
-LOCAL_TO_POLICY_PATCH=$DEFAULT_BRIDGE"-local-to-policy"
-POLICY_TO_LOCAL_PATCH=$DEFAULT_BRIDGE"-policy-to-local"
-POLICY_TO_CLS_PATCH=$DEFAULT_BRIDGE"-policy-to-cls"
-CLS_TO_POLICY_PATCH=$DEFAULT_BRIDGE"-cls-to-policy"
-CLS_TO_UPLINK_PATCH=$DEFAULT_BRIDGE"-cls-to-uplink"
-UPLINK_TO_CLS_PATCH=$DEFAULT_BRIDGE"-uplink-to-cls"
+LOCAL_TO_POLICY_PATCH="${DEFAULT_BRIDGE}-local-to-policy"
+POLICY_TO_LOCAL_PATCH="${POLICY_BRIDGE}-policy-to-local"
+POLICY_TO_CLS_PATCH="${POLICY_BRIDGE}-policy-to-cls"
+CLS_TO_POLICY_PATCH="${CLS_BRIDGE}-cls-to-policy"
+CLS_TO_UPLINK_PATCH="${CLS_BRIDGE}-cls-to-uplink"
+UPLINK_TO_CLS_PATCH="${UPLINK_BRIDGE}-uplink-to-cls"
 
 ovs-vsctl add-br ${DEFAULT_BRIDGE} -- set bridge ${DEFAULT_BRIDGE} protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
-ovs-vsctl add-br ${DEFAULT_BRIDGE}-policy -- set bridge ${DEFAULT_BRIDGE}-policy protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
-ovs-vsctl add-br ${DEFAULT_BRIDGE}-cls -- set bridge ${DEFAULT_BRIDGE}-cls protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
-ovs-vsctl add-br ${DEFAULT_BRIDGE}-uplink -- set bridge ${DEFAULT_BRIDGE}-uplink protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
+ovs-vsctl add-br ${POLICY_BRIDGE} -- set bridge ${POLICY_BRIDGE} protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
+ovs-vsctl add-br ${CLS_BRIDGE} -- set bridge ${CLS_BRIDGE} protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
+ovs-vsctl add-br ${UPLINK_BRIDGE} -- set bridge ${UPLINK_BRIDGE} protocols=OpenFlow10,OpenFlow11,OpenFlow12,OpenFlow13 fail_mode=secure
 
 ip link set ${DEFAULT_BRIDGE} up
-ip link set ${DEFAULT_BRIDGE}-policy up
-ip link set ${DEFAULT_BRIDGE}-cls up
-ip link set ${DEFAULT_BRIDGE}-uplink up
+ip link set ${POLICY_BRIDGE} up
+ip link set ${CLS_BRIDGE} up
+ip link set ${UPLINK_BRIDGE} up
 
 ovs-vsctl \
-    -- add-port $DEFAULT_BRIDGE $LOCAL_TO_POLICY_PATCH \
-    -- set interface $LOCAL_TO_POLICY_PATCH type=patch options:peer=$POLICY_TO_LOCAL_PATCH \
```

Comment:
no need to set ofport here, which detected in datapath automatically.

Reply:
the command won't set the ofport here.

Reply:
Sry, misread it.